### ISSUE TITLE
Accessiblity Issue Colour Contrast Class .red.white-text

### ISSUE THREAD

**Issue by CalvinRodo** [DIS-SIN/Evalhalla#1]
The .red.white-text class that can be seen on the following page:
https://dis-sin.github.io/Evalhalla/?sur=ut1_june18_event

Fails WCAG 1.4.3:
https://dequeuniversity.com/rules/axe/3.2/color-contrast

Here is the link to the WAV Color Contrast Checker:

https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=F44336

The contrast is okay on large text but it's contrast ratio of 3.6.8:1 is not sufficient for WCAG AA as the contrast has to be at least 4.5:1 and since the user can change the rendering size of the font you need to align to the Normal Text Rules.

You can change the inner text to black to be compliant although it still fails AAA but you only need to be AA compliant so not a big deal.

**Comment by mightyweasel** [comment]
Hey Calvin, thanks for the feedback and suggestions.

I've got a refactor commit in the works that should be deployed soon (probably before mid week next), it should address the three items you identified. Here's some additional detail: 
 
For the disabled button:

I think that falls under the following:
https://webaim.org/articles/contrast/

Incidental
WCAG 2.0 defines four types of "incidental" text that are not required to meet the contrast requirements.
Inactive: An inactive element, like a disabled Submit button (  Submit ), is identified visually by its lower-contrast state.

But to live up to the spirit of the guidelines I went ahead and altered the design a bit. Should be more clear that it's disabled even when the page is desaturated.

White text red background:

See other's opinions here. I'd argue luminance and perception here play a role as well.
https://ux.shopify.com/in-plain-sight-5639c9afb4c6
and
https://www.designworkplan.com/read/signage-and-color-contrast

Regardless, I've managed to find a color that still looks red (#DB0F2A), but has a 5.1:1 ratio. 
At 7:1 it lost the "stop sign" effect we were aiming for. We'll definitely revisit if we can get AAA in a future sprint or if we get user feedback that it's still not good enough to be clear.

Optional Labels (grey class):

For the optional labels I darkened the grey to meet AAA contrast (#565656) 7.33:1 ratio.

Cheers and thanks for the interest
